tinlake-ui/services/centChain/isCentChainAddr.ts

```diff
@@ -1,13 +1,12 @@
 import { Keyring } from '@polkadot/api'
 
 export function isCentChainAddr(addr: string) {
```

Comment:
don't think this is even needed. any substrate address will work no matter which chain it's formatted as

Reply:
just `import { isAddress } from '@polkadot/util-crypto'` should do the trick.